src/app/components/cards/PostSummary.jsx

```diff
@@ -383,8 +383,10 @@ export default connect(
         return {
             post,
             content,
-            pending_payout,
-            total_payout,
+            pending_payout: pending_payout
+                ? pending_payout.toString()
+                : pending_payout,
+            total_payout: total_payout ? total_payout.toString() : total_payout,
```

Comment:
What is the purpose of this change? IIRC, both these props, if set, are strings already.

Reply:
It depends where the post comes from. The pinned post I get from `steem-js` has these fields as numbers.

Reply:
Gotcha